Assets/Mirror/Runtime/Transport/Websocket/Plugins/WebSocket.jslib

```diff
@@ -18,14 +18,12 @@ var LibraryWebSockets = {
 		}
 
 		socket.onmessage = function (e) {
-			console.log("received data data= " +e.data);
```

Comment:
While I agree with this 100%,  

Why is this in this pull request?  can you separate it into a different PR?